duckduckgo/duckduckgo-privacy-extension
# Update remaining old icons

**Reviewer:** @laurengarcia 



## Description:
I just noticed while testing the extension, some of the old icons in the repo weren't updated (we didn't have new assets for them), so in the trackers subview they looked broken, because:
1. We see we have the icon available and we try to render it
2. It's the old icon with the grey background and transparent fill, so it just shows up as a solid grey circle, looking broken

Uploaded the new icons in this PR and updated the styles.
We now have a different set for the top offenders pill version, and for the tracker networks subview.
The first are the tn-*.svg icons, the subview ones are the tn-g-*.svg ones (g is for grey).

<img width="304" alt="screen shot 2018-01-04 at 11 24 22 am" src="https://user-images.githubusercontent.com/3652195/34575010-313cfe3a-f147-11e7-9128-d2375cedd8b0.png">



## Steps to test this PR:
1. Build and reload extension
2. Test websites such as huffingtonpost.com, 
4. Pill version is unchanged
5. Tracker networks subview has the grey background icons as expected, and salesforce, openx, criteo and casale show up normally


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
